.npmignore

```diff
@@ -0,0 +1 @@
+unity-tests/
```

Comment:
this prevents the `unity-tests` directory from being downloaded when a user imports our SDK